mera-weather-app/src/components/Main.js

```diff
@@ -117,6 +120,15 @@ const Main = () => {
             );
           })}
       </SevenDays>
+      <SevenDaysNames>
+        {weatherData &&
+          weatherData.daily.slice(0, -1).map((day, key) => {
+            const date = new Date(day.dt * 1000);
+            const weekdayNum = date.getDay();
+
+            return <DayName>{getWeekday(weekdayNum)}</DayName>;
```

Comment:
Just FYI, you don't even need this `getWeekday` helper, the native `Date` object has all the methods you need to get the day name ;)